themes/powerline/powerline.theme.bash

```diff
@@ -2,6 +2,8 @@
 
 . "$BASH_IT/themes/powerline/powerline.base.bash"
 
+CAN_SUDO=true
```

Comment:
Thanks for your comments nwinkler. I didn't understand when you say the user can't override `CAN_SUDO=true`. Just setting `CAN_SUDO=false` does the job, I tested here in my environment. But I have indeed see this kind of construct before in Bash scripts where people set variables the way `THEME_CHECK_SUDO=${THEME_CHECK_SUDO:=true}` although I don't fully understand the benefits of it entirely.